projects/1.19/assets/sweetmagic/sweetmagic/lang/zh_cn.json

```
@@ -0,0 +1,1097 @@
+	"item.sweetmagic.magic_flamenova": "烈焰新星",
+	"item.sweetmagic.magic_frost": "霜击",
+	"item.sweetmagic.magic_frostspear": "霜矛",
+	"item.sweetmagic.magic_tornado": "风旋",
+	"item.sweetmagic.magic_storm": "风爆",
+	"item.sweetmagic.magic_burst": "爆裂",
+	"item.sweetmagic.magic_blast": "爆燃",
+	"item.sweetmagic.magic_ballast": "重力爆破",
+	"item.sweetmagic.magic_gravitywave": "引力势阱",
+	"item.sweetmagic.magic_poison_shoot": "毒针",
+	"item.sweetmagic.magic_toxic": "毒云",
+	"item.sweetmagic.magic_lightningbolt": "雷击",
+	"item.sweetmagic.magic_thunder": "雷暴",
+	"item.sweetmagic.magic_dig": "采掘",
+	"item.sweetmagic.magic_silk_dig": "精准采掘",
+	"item.sweetmagic.magic_normal": "原型射击",
+	"item.sweetmagic.magic_completion": "强化射击",
+	"item.sweetmagic.magic_aether_armor": "以太装甲",
+	"item.sweetmagic.magic_aether_barrier": "以太屏障",
+	"item.sweetmagic.magic_effectremover": "净化",
+	"item.sweetmagic.magic_reflasheffect": "涤净",
+	"item.sweetmagic.magic_growth_effect": "生长",
+	"item.sweetmagic.magic_growth_aura": "疯长",
+	"item.sweetmagic.magic_bubleprison": "气泡囚牢",
+	"item.sweetmagic.magic_scumefang": "溺于气泡",
+	"item.sweetmagic.magic_bloodshot": "鲜血弹",
+	"item.sweetmagic.magic_bloodwave": "鲜血冲击",
+	"item.sweetmagic.magic_water_breath": "水下呼吸",
+	"item.sweetmagic.magic_night_vision": "夜视",
+	"item.sweetmagic.magic_speed": "迅捷",
+	"item.sweetmagic.magic_strength": "力量",
+	"item.sweetmagic.magic_book": "不可思议的魔术书「甜蜜魔法」",
+	"item.sweetmagic.magic_book_cosmic": "不可思议的魔术书「宇宙魔法」",
+	"item.sweetmagic.aether_crystal_shard": "以太水晶碎片",
+	"item.sweetmagic.aether_crystal": "以太水晶",
+	"item.sweetmagic.divine_crystal": "神圣水晶",
+	"item.sweetmagic.pure_crystal": "纯净水晶",
+	"item.sweetmagic.electronic_orb": "球状闪电",
+	"item.sweetmagic.grav_powder": "引力粉尘",
+	"item.sweetmagic.poison_bottle": "毒液小瓶",
+	"item.sweetmagic.stray_soul": "徘徊之魂",
+	"item.sweetmagic.tiny_feather": "微风之羽",
+	"item.sweetmagic.unmeltable_ice": "不融之冰",
+	"item.sweetmagic.witch_tears": "巫女之泪",
+	"item.sweetmagic.mf_stuff": "魔力御流之杖",
+	"item.sweetmagic.b_mf_bottle": "魔力空瓶",
+	"item.sweetmagic.b_magia_bottle": "巫师魔力空瓶",
+	"item.sweetmagic.mf_small_bottle": "魔力小瓶",
+	"item.sweetmagic.mf_bottle": "魔力瓶",
+	"item.sweetmagic.magia_bottle": "巫师魔力瓶",
+	"item.sweetmagic.blank_page": "空白书页",
+	"item.sweetmagic.blank_magic": "魔力石板",
+	"item.sweetmagic.ender_shard": "末影碎片",
+	"item.sweetmagic.magic_meal": "魔力粉尘",
+	"item.sweetmagic.milk_pack": "牛奶盒",
+	"item.sweetmagic.watercup": "水杯",
+	"item.sweetmagic.raspberry_seed": "树莓",
+	"item.sweetmagic.prizmium": "棱光花瓣",
+	"item.sweetmagic.mysterious_page": "神秘书页",
+	"item.sweetmagic.mystical_page": "魔力书页",
+	"item.sweetmagic.fire_powder": "烈焰尘",
+	"item.sweetmagic.plant_chips": "植物碎屑",
+	"item.sweetmagic.cotton_cloth": "棉布",
+	"item.sweetmagic.seed_bag": "种子袋",
+	"item.sweetmagic.egg_bag": "鸡蛋袋",
+	"item.sweetmagic.acce_bag": "饰品袋",
+	"item.sweetmagic.olive_seed": "橄榄",
+	"item.sweetmagic.gelatin": "明胶",
+	"item.sweetmagic.rice_plants": "水稻",
+	"item.sweetmagic.rice_seed": "水稻种子",
```

```suggestion
	"item.sweetmagic.rice_seed": "稻穗",
```
仅作建议，具体看贴图
反正镇守府和urushi同一个词是分别对应`水稻种子`和`稻穗`的（